test/Generator/ValueGeneratorTest.php

```diff
@@ -115,6 +129,54 @@ public function testPropertyDefaultValueCanHandleComplexArrayOfTypes()
         $this->assertEquals($expectedSource, $generatedTargetSource);
     }
 
+    public function testPropertyDefaultValueCanHandleComplexShortArrayOfTypes()
+    {
+        $targetValue = [
+            5,
+            'one' => 1,
+            'two' => '2',
+            'constant1' => '__DIR__ . \'/anydir1/anydir2\'',
```

Comment:
Well, I just copied the existing test case which had single quotes there ;-). But will fix both tests.